test/__mocks__/audio-effects.js

```diff
@@ -14,7 +14,10 @@ export default class MockAudioEffects {
         this.buffer = buffer;
         this.name = name;
         this.process = jest.fn(done => {
-            this._finishProcessing = renderedBuffer => done(renderedBuffer, 0, 1);
+            this._finishProcessing = renderedBuffer => {
+                done(renderedBuffer, 0, 1);
+                return new Promise(resolve => setTimeout(resolve));
```

Comment:
Now that the rest submitNewSamples pipeline is async, this was needed to allow the tests to be run using async/await

Reply:
and it doesn't work without the setTimeout, like 

```
                return new Promise(resolve => { resolve(); }));
```
? Interesting/surprising!

Reply:
Yeah unfortunately because there are several promises chained